include/ignition/math/RollingMean.hh

```diff
@@ -18,6 +18,7 @@
 #define IGNITION_MATH_ROLLINGMEAN_HH_
 
 #include <memory>
+#include <limits>
```

Comment:
Out of scope for this PR?

Reply:
I agree and will remove from the PR, only put this library because before it the build is giving the following error:

`
ign-math/src/RollingMean.cc:57:10: error: 'numeric_limits' is not a member of 'std'
` 

Must I create an Issue for this?

Reply:
It's fine to put in here, but since it's used in the cc file, rather than the header, it should be added there, I believe.